interface/billing/sl_eob_patient_note.php

```diff
@@ -24,68 +24,69 @@
 ?>
 <html>
 <head>
-<?php Header::setupHeader();?>
-<title><?php echo xlt('EOB Posting - Patient Note'); ?></title>
+    <?php Header::setupHeader(['opener']); ?>
+    <title><?php echo xlt('EOB Posting - Patient Note'); ?></title>
 </head>
 <body>
-<?php
-  $patient_id = $_GET['patient_id'];
-if (! $patient_id) {
-    die(xlt("You cannot access this page directly."));
-}
+    <?php
 
-if ($_POST['form_save']) {
-    if (!CsrfUtils::verifyCsrfToken($_POST["csrf_token_form"])) {
-        CsrfUtils::csrfNotVerified();
+    $patient_id = $_GET['patient_id'];
+    if (!$patient_id) {
+        die(xlt("You cannot access this page directly."));
     }
 
-    $thevalue = trim($_POST['form_note']);
+    $row = sqlQueryNoLog("Select billing_note From patient_data Where pid=?", [paitient_id]);
```

Comment:
Is `[paitient_id]` correct?
Also, this is a place where should prob not bypass the logging.
(goal is for logging to be faster than no logging; just need a couple more log optimizations and reversal of several basic physic laws :) )

Reply:
> `[paitient_id]` correct?

no, how the heck did it even work?
logging a billing note, aww, don't care either way.

Reply:
btw, reworking logging (ie. shifting to a mechanism that doesn't dramatically impact performance) could be the next big project after:
1. bootstrap
2. sessions
3. log